plugins/hosts/bsd/cap/nfs.rb

```diff
@@ -192,6 +211,18 @@ def self.nfs_checkexports!
             raise Vagrant::Errors::NFSBadExports, output: r.stderr
           end
         end
+
+        def self.nfs_check_folders_for_apfs(folders)
+          folders.each do |_, opts|
+            # check to see if this path is mounted in an APFS filesystem, and if it's in the
+            # firmlink which must be prefixed.
+            is_mounted_apfs_command = "df -t apfs #{opts[:hostpath]}"
```

Comment:
`is_mounted_apfs_command = "df -t apfs #{opts[:hostpath]}"` -> `is_mounted_apfs_command = "/bin/df -t apfs #{opts[:hostpath]}"` 
per above.

Reply:
Ah, that's the problem — I've committed the change. Thanks!

Reply:
LGTM && thanks!